proposals/csharp-10.0/parameterless-struct-constructors.md

```diff
@@ -0,0 +1,157 @@
+The struct can be declared as `struct`, `readonly struct`, or `ref struct`.
+
+If the struct does not declare a parameterless instance constructor, and the struct has no fields with variable initializers, the struct (see [struct constructors](https://github.com/dotnet/csharplang/blob/master/spec/structs.md#constructors)) ...
+> implicitly has a parameterless instance constructor which always returns the value that results from setting all value type fields to their default value and all reference type fields to null.
+
+If the struct does not declare a parameterless instance constructor, and the struct has field initializers, a `public` parameterless instance constructor is synthesized.
+The parameterless constructor is synthesized even if all initializer values are zeros.
+
+### Modifiers
+A parameterless instance constructor may be less accessible than the struct.
+
+A parameterless instance constructor may be declared `extern`.
+
+### Executing field initializers
+Execution of struct instance field initializers matches execution of [class field initializers](https://github.com/dotnet/csharplang/blob/master/spec/classes.md#instance-variable-initializers):
+> When an instance constructor has no constructor initializer, ... that constructor implicitly performs the initializations specified by the _variable_initializers_ of the instance fields ... . This corresponds to a sequence of assignments that are executed immediately upon entry to the constructor and before the implicit invocation of the direct base class constructor. The variable initializers are executed in the textual order in which they appear in the ... declaration.
+
+### Definite assignment
+Instance fields must be definitely assigned in struct instance constructors that do not have a `this()` initializer (see [struct constructors](https://github.com/dotnet/csharplang/blob/master/spec/structs.md#constructors)).
+
+Definite assignment is required for explicit parameterless constructors as well.
+Definite assignment _is not required_ for synthesized parameterless constructors.
+```csharp
+struct S0
+{
+    object x = null;
+    object y;
+    // ok
+}
+
+struct S1
+{
+    object x = null;
+    object y;
+    S() { } // error: field 'y' must be assigned
+}
+
+struct S2
+{
+    object x = null;
+    object y;
+    S() : this(null) { }        // ok
+    S(object y) { this.y = y; } // ok
+}
+```
+
+_The synthesized parameterless constructor may need to emit `ldarg.0 initobj S` before any field initializers. Describe the specifics. How is this handled in Visual Basic?_
+
+### No `base()` initializer
+A `base()` initializer is disallowed in struct constructors.
+
+The compiler will not emit a call to the base `System.ValueType` constructor for any struct instance constructors including explicit and synthesized parameterless constructors.
+
+### Constructor use
+
+The parameterless constructor may be less accessible than the containing value type.
```

Comment:
Can we make the accessibility of the `struct` themselves explicit here just to drive home the sample?